src/index.ts

```diff
@@ -4,3 +4,4 @@ export * from './general';
 export * from './jsx';
 export * from './string';
 export * from './number';
+export * from './number/NumberFormatter/numberConfig';
```

Comment:
Can you comment here that ...
Ideally, numberConfig should be exported from NumberFormatter file but it generates the doc which we don't want and that is why it is exported from here.